api/v1/cosmosfullnode_types.go

```diff
@@ -32,11 +32,21 @@ const CosmosFullNodeController = "CosmosFullNode"
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+// Ordinal specifies the configuration for pod ordinal numbers
+type Ordinal struct {
+	// Start specifies the initial ordinal number for pod naming
+	// +kubebuilder:validation:Minimum:=0
+	Start int32 `json:"start,omitempty"`
+}
+
 // FullNodeSpec defines the desired state of CosmosFullNode
 type FullNodeSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
+	// Ordinal specifies the configuration for pod ordinal numbers
```

Comment:
From StatefulSet

> ordinals controls the numbering of replica indices in a StatefulSet. The default ordinals behavior assigns a "0" index to the first replica and increments the index by one for each additional replica requested.

Reply:
Source: https://www.manifests.io/kubernetes/1.31/io.k8s.api.apps.v1.StatefulSetOrdinals?linked=StatefulSet.spec.ordinals

Reply:
Updated such that we have it as `spec.ordinals.start` instead of  `spec.ordinal.start`.

Reply:
Updated descriptions appropriately in sturct and autogenerated file.